targets/f7/furi_hal/furi_hal_interrupt.h

```diff
@@ -53,27 +53,46 @@ typedef enum {
     FuriHalInterruptIdMax,
 } FuriHalInterruptId;
 
+typedef enum {
+    FuriHalInterruptPriorityVeryLow =
+        -2, /**< Lowest priority level, you can use ISR-safe OS primitives */
+    FuriHalInterruptPriorityLow =
+        -1, /**< Lower than normal priority level, you can use ISR-safe OS primitives */
+    FuriHalInterruptPriorityNormal =
+        0, /**< Normal(default) priority level, you can use ISR-safe OS primitives */
+    FuriHalInterruptPriorityHigh =
+        1, /**< Higher than normal priority level, you can use ISR-safe OS primitives */
+    FuriHalInterruptPriorityVeryHigh =
+        2, /**< Highest priority level, you can use ISR-safe OS primitives */
+
+    /* Special group, read docs first(ALL OF THEM: especially FreeRTOS configLIBRARY_MAX_SYSCALL_INTERRUPT_PRIORITY) */
+    FuriHalInterruptPriorityKamiSama =
+        3, /**< Forget about thread safety, you are god now, no one can prevent you from fucking up OS critical section, you are not allowed to use any OS primitives, but who can stop you? Use this priority only for direct hardware interaction with LL HAL. */
+} FuriHalInterruptPriority;
+
 /** Initialize interrupt subsystem */
 void furi_hal_interrupt_init();
 
-/** Set ISR and enable interrupt with default priority
- * We don't clear interrupt flags for you, do it by your self.
- * @param index - interrupt ID
- * @param isr - your interrupt service routine or use NULL to clear
- * @param context - isr context
+/** Set ISR and enable interrupt with default priority We don't clear interrupt
```

Comment:
Suggesting more neutral wording:
"Interrupt flags are not cleared automatically, the user code must do it by calling ..."